excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/CategoryParserTest.java

```diff
@@ -59,6 +59,7 @@ public void testParse() {
         assertThat(categoryEntities.get(0).getCategoryLabel(), is(CATEGORY_LABEL));
         assertThat(categoryEntities.get(0).getDisplayOrder(), is(DISPLAY_ORDER));
         assertThat(categoryEntities.get(0).getParentCategoryId(), is(NO_PARENT));
+        assertThat(parseContext.getCategory(CASE_TYPE, CATEGORY_ID), is(categoryEntities.get(0)));
```

Comment:
use import

`import static org.hamcrest.MatcherAssert.assertThat;`

to get rid of deprecation warnings?